### ISSUE TITLE
Clicked on a post when bottom is reached

### ISSUE THREAD

**Issue by l3d00m** [ccrama/Slide#153]
## Info
1. Quickly scroll to bottom of a sub
2. Click the last post at bottom before new posts are loaded.  
3. Post is loading and then slide crashes. 

## Stacktrace
```
java.lang.IllegalStateException: The application's PagerAdapter changed the adapter's contents without calling PagerAdapter

**Comment by ccrama** [comment]
What post did you click on? Was it from  the subreddit view?

**Comment by l3d00m** [comment]
I was scrolling though the frontpage and clicked on one post from /r/imgoingtohellforthis and it crashed directly. I cannot reproduce it (I think [this](http://reddit.com/r/ImGoingToHellForThis/comments/3p87ct/the_struggle_is_real/) one?)

**Comment by l3d00m** [comment]
It happened a second time and I now know how to repro.  
1. Close app so that it reloads at next startup  
2. start app
3. Quickly scroll to bottom of frontpage (I'm in single mode, dunno if that's relevant)  
4. Click the last post at bottom before new posts are loaded.  
5. Post is loading and then slide crashes.

**Comment by TomerRosenfeld** [comment]
Here is the full exception
``` 
E/AndroidRuntime: java.lang.IllegalStateException: The application's PagerAdapter changed the adapter's contents without calling PagerAdapter#notifyDataSetChanged! Expected adapter item count: 100, found: 200 Pager id: me.ccrama.redditslide:id/contentView Pager class: class android.support.v4.view.ViewPager Problematic adapter: class me.ccrama.redditslide.Activities.CommentsScreen$OverviewPagerAdapter
```

**Comment by l3d00m** [comment]
Marking this high priority because it's happening pretty often and is related to some other issues (#765 for example)

**Comment by l3d00m** [comment]
Fixed with #818, thank you very much @Deadleg for your great pr!
